tgui/packages/tgui/interfaces/GeneTek/AppearanceEditor.js

```diff
@@ -127,7 +127,7 @@ export const AppearanceEditor = (params, context) => {
             }}
             style={{
               width: "80px",
-              height: "80px",
+              height: "160px",
```

Comment:
Sanity check on whether this causes the image to be stretched a little bit? Would making this 64x128 be better?

Reply:
Most likely it would be better but since original was 80px I didn't want to change it as a part of this.